tools/tests/dropdown/multiple-dropdown.test.js

```diff
@@ -257,4 +269,25 @@ describe('Multiple Dropdown Test', () => {
             });
         });
     });
+
+
+    describe('Multiple Dropdown Component (Disabled Options)', () => {
+        beforeEach(function (done) {
+            setupMultipleDropdownTestPage(multipleDropdownWithDisabledOptionsTemplate)
+                .then(done)
+                .catch(err => console.error(err));
+        });
+
+        it('Should select multiple elements using shift key and click', () => {
+            const dropdownWrapper = document.querySelector('.multiple-dropdown-test-wrapper');
+            const dropdown = dropdownWrapper.querySelector('gameface-dropdown');
+            const options = dropdown.querySelectorAll('dropdown-option');
+
+            dropdown.onClickOption(mockEventObject(options[2], true, true, true));
+            dropdown.onClickOption(mockEventObject(options[6], true, true, true));

```

Comment:
`mockEventObject` has `ctrlKey` param as the last one and here it will send an event with a pressed ctrl key instead of a shift. Rework the `mockEventObject` to receive the `shiftKey` param and the tests may pass.

Reply:
Also the `mockEventObject` method currently has 3 params and not 4.

Reply:
Yep, apparently I forgot to commit actions.js